main/inc/ajax/model.ajax.php

```diff
@@ -174,20 +174,22 @@ function getWhereClause($col, $oper, $val)
                 // Extra field.
                 $extraField = new ExtraField($type);
 
-                foreach ($filters->rules as $key => $data) {
-                    if (empty($data)) {
-                        continue;
-                    }
-                    if ($data->field === 'extra_access_start_date') {
-                        $accessStartDate = $data->data;
-                    }
-
-                    if ($data->field === 'extra_access_end_date') {
-                        $accessEndDate = $data->data;
-                    }
-
-                    if (in_array($data->field, $toRemove)) {
-                        unset($filters->rules[$key]);
+                if (is_object($filters) && property_exists($filters, 'rules') && is_array($filters->rules) && !empty($filters->rules)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 136 characters](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4663904)